projects/1.20/assets/ruins/philipsruins/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+    "block.philipsruins.red_sand_stone_brick": "红砂岩砖",
+    "block.philipsruins.sand_stone_brick": "砂岩砖",
+    "block.philipsruins.red_sand_stone_tiles": "红砂岩瓷砖",

```

```suggestion
    "block.philipsruins.red_sand_stone_tiles": "红砂岩瓦",
```
这个方块的材质是什么样的
见原版`深板岩瓦`/`Deepslate Tiles`